cypress.config.js

```diff
@@ -19,6 +19,41 @@ module.exports = defineConfig({
             password: '12345Qwert!'
           };
         },
+        settingsData() {
+          return {
+            bio: 'New bio information',
+            newEmail: 'test@qa.team',
+            newPassword: 'Test1234!',
+            email: 'riot@qa.team',
+            username: 'riot',
+          };
+        },
+        signInData() {
+          return {
+            notExistEmail: 'test@gmail.com',
+            notExistPassword: 'Qatest12',
+            passwordRequiredMessage: 'Password field required.',
+            emailRequiredMessage: 'Email field required.',
+            invalidCredentialsMessage: 'Invalid user credentials.',
+          };
+        },
+        signUpData() {
```

Comment:
I'm not sure, that those methods should be in a config file. It is better to create a separate folder in the support folder to keep this methods